.github/workflows/decision.yml

```diff
@@ -0,0 +1,80 @@
+name: Create Branch and PR from Issue Label
+
+on:
+  issues:
+    types: [labeled]
+
+jobs:
+  create_branch_and_pr:
+    if: github.event.label.name == 'decision'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+
+    steps:
+    - name: Authorization
+      run: |
+        # Get username of GitHub account that triggered this flow
+        opener=${{ github.actor }}
+        
+        # Get list of approved contributors
+        contributors=$(curl https://api.github.com/repos/Monstarrrr/Rebutify/contributors | jq -r '[.[] | select(.type == "User") | .login] | join(",")')
+        
+        # Convert the comma-separated string to an array
+        IFS=',' read -r -a contributor_array <<< "$contributors"
+
+        # Check if triggering user is in the array
+        if [[ ! " ${contributor_array[@]} " =~ " $opener " ]]; then
+          echo "Triggering user is unauthorized."
+          exit 1
+        fi
+    - name: Check out code
+      uses: actions/checkout@v3
+
+    - name: Extract subject and category from issue title
+      id: extract_info
+      run: |
+        title="${{ github.event.issue.title }}"
+        if [[ "$title" =~ ^Opening\ decision\ on\ ([^/]+)(/([^/]+))?$ ]]; then
+          subject=$(echo "${BASH_REMATCH[1]}" | tr -d ' ')
+          category=$(echo "${BASH_REMATCH[3]}" | tr -d ' ')
+          if [[ -z "$category" ]]; then
+            category=""
+          fi
+          echo "::set-output name=subject::$subject"
+          echo "::set-output name=category::$category"
+        else
+          echo "Issue title does not follow the required format: 'Opening decision on [category]/[subject]'"
+          exit 1
+        fi
```

Comment:
> The point of this flow is to inject code into the repo.

> That's what the security warning doesn't like.

> Due to the authorization step (only commits if triggering user is contributor) this is not a concern since contributors can already inject code via git by pushing.  

> @seporterfield